tests/ui-api-layer-acceptance-tests/README.md

```diff
@@ -38,13 +33,13 @@ Run acceptance tests using the following command:
 - against standalone UI API Layer deployed on the local host:
   
   ```bash
-  KUBE_CONFIG=/Users/${USER}/.kube/config IS_LOCAL_CLUSTER=false GRAPHQL_ENDPOINT=http://localhost:3000/graphql go test ./... -tags=acceptance
+  KUBE_CONFIG=/Users/${USER}/.kube/config IS_LOCAL_CLUSTER=false GRAPHQL_ENDPOINT=http://localhost:3000/graphql USERNAME=admin@kyma.cx PASSWORD=$(kubectl get secret admin-user -n kyma-system -o jsonpath="{.data.password}" | base64 -D) go test ./... -tags=acceptance
```

Comment:
where do you use the `USERNAME` and `PASSWORD`?

Reply:
E.g. here: https://github.com/kyma-project/kyma/blob/master/tests/ui-api-layer-acceptance-tests/graphql/config.go#L77